builds/jazz-build-module/utility-loader.groovy

```diff
@@ -225,14 +225,25 @@ def constructArn (arn, resourceName, resourceType, config) {
 	return arn
 }
 
-def getStackResources (stackName, region, credsId) {	
-	def stackResources = sh(script: "aws cloudformation describe-stack-resources --stack-name ${stackName} --region ${region} --profile ${credsId}", returnStdout: true)
-	echo "Describe Stacks are ${stackResources}"
-	def parsedResources = parseJson(stackResources)
-	return parsedResources
+
+def getStackResources (stackName, region, credsId) {
+	def stackResources = null	
+	try {
+		stackResources = sh(script: "aws cloudformation describe-stack-resources --stack-name ${stackName} --region ${region} --profile ${credsId}  --output json 2<&1", returnStdout: true)
+		echo "Describe Stacks are ${stackResources}"
+		def parsedResources = parseJson(stackResources)
+		return parsedResources
+	} catch (ex) {
+		echo "stack not exists..."
+		try {
+			def resp = sh(script: "aws cloudformation describe-stack-resources --stack-name ${stackName} --region ${region} --profile ${credsId}  --output json 2<&1 | grep -c 'ValidationError'", returnStdout: true)
+			if(resp != 1) error "describe stack failed."
+			else return stackResources
+		} catch (e) {}
+	}
 }
 
-def createAllStackResources (whiteListModule, events, config, stackResources, env) {
+def createAllStackResources (whiteListModule, events, config, stackResources, env, isSendEvent) {
```

Comment:
Can we add comments to clarify what is being done here?

Reply:
`isSendEvent` => `createAsset`

Reply:
When and why would we not send `CREATE_ASSET`?

Reply:
Done

Reply:
We need to send events only after deployment, and we don't need to send the event while we are calling this method only for creating the resource map